classes/PipelineRequest/mock.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Mocked PipelineRequest.
```

Comment:
Please provide an extended description outlining the purpose and basic feature set of this class.